### PR TITLE
Update CNV network policy

### PR DESCRIPTION
**Pull request by rajivnathan**
Adds a selector to enable ingress from openshift-cnv.

### CODE REVIEW

File: deploy/templates/nstemplatetiers/base/ns_stage.yaml [hunk 1]
```diff
@@ -173,14 +173,18 @@ objects:
 - apiVersion: networking.k8s.io/v1
   kind: NetworkPolicy
   metadata:
-    name: allow-from-openshift-virtualization-os-images
-    namespace: ${SPACE_NAME}-stage
+    name: allow-from-openshift-virtualization-namespaces
+    namespace: ${SPACE_NAME}-dev
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/base/ns_stage.yaml:177]:
```suggestion
    namespace: ${SPACE_NAME}-stage
```

**Review by alexeykazakov — COMMENTED** [review]
Looks good. Just one comment to address.

File: deploy/templates/nstemplatetiers/base/ns_stage.yaml (hunk 1 above)
Comment (rajivnathan) [reply deploy/templates/nstemplatetiers/base/ns_stage.yaml]: Oops, nice catch!

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**